schema.sql

```diff
@@ -4,25 +4,30 @@ CREATE DATABASE bookshelf;
 
 USE bookshelf;
 
-CREATE TABLE books (
-  id int not null auto_increment,
-  PRIMARY KEY (id)
-);
+-- CREATE TABLE books (
+--   id int not null auto_increment,
+--   PRIMARY KEY (id)
+-- );
 
 CREATE TABLE users (
   id int not null auto_increment,
-  username varchar(20) not null,
+  username varchar(50) not null,
   PRIMARY KEY (id)
 );
 
 CREATE TABLE reviews (
   id int NOT NULL AUTO_INCREMENT,
-  user_id INT NOT NULL,
-  book_id INT NOT NULL,
+  user_id INT default null,
+  book_id INT default null,
   date date,
   review varchar(1000) NOT NULL,
```

Comment:
I think you might want to make this 2000 characters since 1000 is only 150 words